test/testdata/namer/type_member_redefs.rb

```diff
@@ -0,0 +1,20 @@
+# typed: true
+# disable-fast-path: true
+
+class A
```

Comment:
Can we add a test with `R` defined in multiple files? It should also be an error.

Reply:
This test now includes a second file to verify that the second one is an error. As per online conversation, it still doesn't handle multiple class openings in the same file, but I'll consider how to make that work.